README.md

```diff
@@ -15,6 +15,25 @@ In order for a Docker container to work with this package it must:
 - listen on some port for HTTP connections, and
 - accept a single JSON file as input.
 
+## Quick Demo
+
+Install [Docker](https://store.docker.com/search?offering=community&type=edition)
+if you haven't already, then download the project, install dependencies, and
+run the demo server:
+
+```
+$ git clone https://github.com/mccalluc/django_docker_engine.git
+$ cd django_docker_engine
+$ pip install -r requirements.txt
```

Comment:
This step didn't work for me initially with a clean `python3.6` virtualenv:
```
pip install -r requirements.txt
Looking in indexes: https://pypi.python.org/simple/
Obtaining file:///Users/scott/django_docker_engine (from -r requirements.txt (line 2))
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/Users/scott/django_docker_engine/setup.py", line 6, in <module>
        import yaml
    ModuleNotFoundError: No module named 'yaml'

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /Users/scott/django_docker_engine/
```

but:
``` bash
$ pip install -r requirements-dev.txt
$ pip install -r requirements.txt
``` 
got me up and running

Reply:
Thanks: I'll change the README to match that.